### PR TITLE
feature: Add togglz for work contributors in work summary

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA][my-orcid] New functionality for contributors in works not implemented](https://trello.com/c/t8fa5jir/7704-qamy-orcid-new-functionality-for-contributors-in-works-not-implemented)

File: src/app/record/components/work-stack/work-stack.component.ts [hunk 1]
```diff
@@ -92,8 +101,12 @@ export class WorkStackComponent implements OnInit {
       this.panelDetailsState[work.putCode.value] = {
         state: false,
       }
+    } else {
+      this.getDetails(work).subscribe()
+    }
+    if (this.togglzWorksContributors) {
+      this.getDetails(work).subscribe()
     }
```
Comment (amontenegro) [inline src/app/record/components/work-stack/work-stack.component.ts:109]: Please be sure that the work details are fetched only once.

**Review by amontenegro — CHANGES_REQUESTED** [review]

File: src/app/record/components/work-stack/work-stack.component.ts [hunk 2]
```diff
@@ -93,7 +102,14 @@ export class WorkStackComponent implements OnInit {
...
       }
     }
-    this.getDetails(work).subscribe()
+    // todo @DanielPalafox remove if and togglz once contributors are returned in works page endpoint
+    if (
+      this.togglzWorksContributors ||
+      this.panelDetailsState[work.putCode.value] !== undefined ||
+      !force
+    ) {
+      this.getDetails(work).subscribe()
```
Comment (leomendoza123) [inline src/app/record/components/work-stack/work-stack.component.ts:111]: Should this be more like this? 

```
this.togglzWorksContributors ||
 (this.panelDetailsState[work.putCode.value] !== undefined &&
 !force) 
 ```

Comment (leomendoza123) [reply src/app/record/components/work-stack/work-stack.component.ts:111]: so it does not retrieve details on a force

**Review by leomendoza123 — APPROVED** [review]

**Review by amontenegro — APPROVED** [review]

**Comment by amontenegro** [comment]
[[QA-devs][my-orcid] Loading users with many works is very slow](https://trello.com/c/018mouF0/7782-qa-devsmy-orcid-loading-users-with-many-works-is-very-slow)

**Pull request merged**